src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -2542,7 +2542,9 @@ static struct PyModuleDef cimpl_moduledef = {
 static PyObject *_init_cimpl (void) {
 	PyObject *m;
 
-        PyEval_InitThreads();
+#if PY_VERSION_HEX < 0x03090000
```

Comment:
Seems like it is only needed in 3.6 and earlier, so perhaps change this to `< 0x..3.7.0`

Reply:
I don't know enough about the CPython API to know whether I could get away with that, but that's good to hear, thanks - I'll update it in a second.

Reply:
Should be fine:
https://docs.python.org/3/c-api/init.html#c.PyEval_InitThreads

Reply:
Great, thanks - I've fixed the commit message too.